app/src/org/commcare/dalvik/preferences/CommCarePreferences.java

```diff
@@ -230,10 +230,15 @@ public static boolean isFuzzySearchEnabled() {
         return properties.getString(FUZZY_SEARCH, NO).equals(YES);
     }
 
+    /**
+     * @return How many seconds should a user session remain open before
+     *         expiring?
+     */
     public static int getLoginDuration() {
         SharedPreferences properties = CommCareApplication._().getCurrentApp().getAppPreferences();
 
-        return properties.getInt(LOGIN_DURATION, 24);
+        // default to 24 hours
+        return Integer.parseInt(properties.getString(LOGIN_DURATION, "28800"));
```

Comment:
Preferences set in HQ come in as strings, so parse into integers. This assumption is okay, right?